### PR TITLE
REPL docs for __init__

### PR DESCRIPTION
**Pull request by Moelf**
Address https://github.com/JuliaLang/julia/issues/28730

### CODE REVIEW

File: base/docs/basedocs.jl [hunk 1]
```diff
@@ -100,6 +100,30 @@ end
 """
 kw"module"
 
+
+    """
+    __init__
+
+`__init_()` function in your module would executes immediately *after* the module is loaded at
```
Comment (giordano) [inline base/docs/basedocs.jl]: Missing an underscore.

Comment (Moelf) [reply base/docs/basedocs.jl]: nice catch. fixed!

**Comment by Moelf** [comment]
not sure why the build test failed.

**Comment by StefanKarpinski** [comment]
@vtjnash, would you mind reviewing the text of this doc string and approving if it looks factually accurate?

**Review by vtjnash — APPROVED** [review]

**Pull request merged**